snuba/settings/__init__.py

```diff
@@ -118,6 +134,25 @@
 REDIS_INIT_MAX_RETRIES = 3
 REDIS_REINITIALIZE_STEPS = 10
 
+T = TypeVar("T")
+
+
+class RedisClusters(TypedDict):
+    cache: RedisClusterConfig | None
+    rate_limiter: RedisClusterConfig | None
+    subscription_store: RedisClusterConfig | None
+    replacements_store: RedisClusterConfig | None
+    misc: RedisClusterConfig | None
+
+
+REDIS_CLUSTERS: RedisClusters = {
+    "cache": None,
```

Comment:
I think we should have these keys be in an Enum, to avoid raw strings being passed into `get_redis_client`.

Reply:
are you talking about the interface to `get_redis_client` here, or would you prefer to configure things with an enum as key? the strings are still typed using `Literal`.